Makefile

```diff
@@ -60,13 +60,15 @@ crd:
 		--volume $(shell pwd):/go/src/github.com/digitalocean/netbox-ip-controller \
 		--volume $(shell go env GOCACHE):/.cache/go-build \
 		golang:${GO_VERSION} bash -c "\
-			git clone --depth 1 --branch kubernetes-${K8S_VERSION} https://github.com/kubernetes/apimachinery /go/src/k8s.io/apimachinery && \
-			git clone --depth 1 --branch kubernetes-${K8S_VERSION} https://github.com/kubernetes/code-generator /go/src/k8s.io/code-generator && \
-			cd /go/src/k8s.io/code-generator && \
+			cd /go/src/github.com/digitalocean/netbox-ip-controller && \
```

Comment:
b/c of https://github.com/kubernetes/code-generator/issues/131, cloning code-generator as a separate go module stopped working
It's supposed to keep working when executed from the root of a module which _vendors_ code-generator, but, when actually vendoring (with go mod commands), some critical subdirectories (`/hack`) are not copied b/c they don't contain any go files necessary for building the go package. So that's why this hack with manually cloning to the `vendor` directory